src/pybind/mgr/cephadm/inventory.py

```diff
@@ -971,7 +989,7 @@ def get_non_draining_hosts(self) -> List[HostSpec]:
         no daemon refresh
         """
         return [
-            h for h in self.mgr.inventory.all_specs() if '_no_schedule' not in h.labels
```

Comment:
nit: consider upadating the comment (we shouldn't make reference to the `_no_schedule` instead we should use the `SpecialHostLabels.drain_daemons_label`